docs/content/server/new_blocks.md

```diff
@@ -84,7 +84,7 @@ Follow these steps to create and test a new block:
 5. **Implement the `run` method with error handling:**, this should contain the main logic of the block:
 

```

Comment:
> For the GitHub integration to work, the back end now needs GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET

In the docs and readme, can we add a section for "integration blocks" or wtv we are calling them and starting adding this kind of information about configuring them?

Reply:
Done: https://github.com/Significant-Gravitas/AutoGPT/pull/8044/commits/04143dbc1aecb5669a7fbecfb23f5ed63c505a77

Reply:
I didn't update the README because:
- I want to avoid duplicating documentation
- The extra stuff for credentials technically fits within the summary instructions for adding blocks that's currently in the README

Reply:
thanks - what about also adding somewhere - where I go to find client id and secret and API key? 
I was thinking we'd just have a subsection somewhere in docs for each integration that just tells you where to go

Reply:
cause rn I can't find exactly where to go for the gh client id and secret....

Reply:
Added to `.env.example`: https://github.com/Significant-Gravitas/AutoGPT/pull/8044/commits/57afc85ad904f1fdef95292d05af41d46b9e14d0